### PR TITLE
[virt] DPDK Checkup: Update image tags

### PR DESCRIPTION
**Pull request by orelmisan**
Update the following tags:
Checkup: v{product-version}.0
Traffic-gen containerDisk image: v0.2.0
VM-under-test containerDisk image: v0.2.0

Version(s):

4.14+

Issue:

Link to docs preview:

https://64283--docspreview.netlify.app/openshift-enterprise/latest/virt/monitoring/virt-running-cluster-checkups#virt-checking-cluster-dpdk-readiness_virt-running-cluster-checkups

QE review:
- [X] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by RamLavi — APPROVED** [review]
/lgtm

File: modules/virt-checking-cluster-dpdk-readiness.adoc [hunk 1]
```diff
@@ -137,7 +137,7 @@ spec:
       restartPolicy: Never
       containers:
         - name: dpdk-checkup
-          image: registry.redhat.io/container-native-virtualization/kubevirt-dpdk-checkup-rhel9:v4.13.0
+          image: registry.redhat.io/container-native-virtualization/kubevirt-dpdk-checkup-rhel9:v{product-version}.0
```
[the remark this answers is not in the document]
Comment (orelmisan) [reply modules/virt-checking-cluster-dpdk-readiness.adoc:140]: It was copied from the VM latency checkup's [documentation](https://github.com/openshift/openshift-docs/blob/ea581c72e4cf1ec78e24c2296d8dd7ce59e43a56/modules/virt-measuring-latency-vm-secondary-network.adoc?plain=1#L147).

[the remark this answers is not in the document]
Comment (sjhala-ccs) [reply modules/virt-checking-cluster-dpdk-readiness.adoc:140]: You need to update the `[source,yaml]` metadata block at the top of the code block (line 126) to `[source,yaml,subs="attributes+"]` for the `{product-version}` attribute to render correctly.

**Review by sjhala-ccs — COMMENTED** [review]
One nit, otherwise LGTM

File: modules/virt-checking-cluster-dpdk-readiness.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (sjhala-ccs) [reply modules/virt-checking-cluster-dpdk-readiness.adoc:140]: FYI, after you update the metadata, the attribute will render as `vBranch Build.0` in the preview, but will resolve to `v4.14.0` in the 4.14 documentation.

[the remark this answers is not in the document]
Comment (orelmisan) [reply modules/virt-checking-cluster-dpdk-readiness.adoc:140]: Thank you @sjhala-ccs.
Done.

**Comment by orelmisan** [comment]
[Change](https://github.com/openshift/openshift-docs/compare/b6bf31da85707796fa14d3cf163ecdf421490b5a..d8fc84a933bb8fc5061ea0e20bfbadd1f1ad11af): Fixed review [comment](https://github.com/openshift/openshift-docs/pull/64283#discussion_r1316065509).

**Pull request merged**